specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -3535,6 +3768,133 @@
       },
       "description": "Filters limit replication to a subset of blobs within the storage account. A logical OR is performed on values in the filter. If multiple filters are defined, a logical AND is performed on all filters."
     },
+    "LocalUsers": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalUser"
+          },
+          "description": "The local users associated with the storage account."
+        }
+      },
+      "description": "List storage account local users."
+    },
+    "LocalUser": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LocalUserProperties",
+          "x-ms-client-flatten": true,
+          "description": "Returns the storage account local user properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The local user associated with the storage accounts."
+    },
+    "LocalUserProperties": {
+      "properties": {
+        "permissionScopes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PermissionScope"
+          },
+          "description": "The permission scopes of the local user."
+        },
+        "homeDirectory": {
+          "type": "string",
+          "description": "Optional, local user home directory."
+        },
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Auto generated by the server for SMB."
+        },
+        "sharedKey": {
+          "$ref": "#/definitions/SharedKey"
+        },
+        "hasSharedKey": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Indicates whether shared key exists."
+        },
+        "hasSshKey": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Indicates whether ssh key exists."
+        }
+      },
+      "description": "The Storage Account Local User properties."
+    },
+    "LocalUserKeys": {
+      "properties": {
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sharedKey": {
+          "$ref": "#/definitions/SharedKey"
+        }
+      },
+      "description": "The Storage Account Local User keys."
+    },
+    "PermissionScope": {
+      "properties": {
+        "permissions": {
```

Comment:
The enum name "permissions" defined here is duplicated with the [before "permissions" enum defined in same file](https://github.com/Azure/azure-rest-api-specs/blob/34543590a641ec448e0a73c27cb250cd6dc2a417/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json#L2982). 
This will make SDK generate fail. Please modify the enum to another name to fix this, before merge this PR.

Reply:
Thanks. Removed the enum part. It will be modeled as a string.